### ISSUE TITLE
jikanpy-4.2.1.tar.gz on pypi lacks CHANGELOG.md which crashes setup.py

### ISSUE THREAD

**Issue by slashbeast** [abhinavk99/jikanpy#81]
The uploaded source package cannot be used to install jikanpy due to missing CHANGELOG.md

```
% pip install https://files.pythonhosted.org/packages/32/d9/da887992e0ee3279a10a06e17aa2aa8f7300887f65f83c51fb87f114b50c/jikanpy-4.2.1.tar.gz
Collecting https://files.pythonhosted.org/packages/32/d9/da887992e0ee3279a10a06e17aa2aa8f7300887f65f83c51fb87f114b50c/jikanpy-4.2.1.tar.gz
  Downloading jikanpy-4.2.1.tar.gz (15 kB)
    ERROR: Command errored out with exit status 1:
     command: /home/piotr/tmp/venv/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/home/piotr/tmp/pip-req-build-m8343il3/setup.py'"'"'; __file__='"'"'/home/piotr/tmp/pip-req-build-m8343il3/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /home/piotr/tmp/pip-pip-egg-info-ikx_1ozb
         cwd: /home/piotr/tmp/pip-req-build-m8343il3/
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/home/piotr/tmp/pip-req-build-m8343il3/setup.py", line 6, in <module>
        CHANGELOG = (HERE / "CHANGELOG.md").read_text()
      File "/usr/lib/python3.8/pathlib.py", line 1232, in read_text
        with self.open(mode='r', encoding=encoding, errors=errors) as f:
      File "/usr/lib/python3.8/pathlib.py", line 1218, in open
        return io.open(self, mode, buffering, encoding, errors, newline,
      File "/usr/lib/python3.8/pathlib.py", line 1074, in _opener
        return self._accessor.open(self, flags, mode)
    FileNotFoundError: [Errno 2] No such file or directory: '/home/piotr/tmp/pip-req-build-m8343il3/CHANGELOG.md'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by abhinavk99** [comment]
Thanks for letting me know. I'll fix this.

**Comment by abhinavk99** [comment]
I'll still fix this, but I'm just curious. Is there a reason you're installing it this way instead of running `pip install jikanpy`? I didn't know people did that and I think it would be good for me to learn why.

**Comment by abhinavk99** [comment]
I just released [v4.2.2 on PyPI](https://pypi.org/project/jikanpy) so try running `pip install https://files.pythonhosted.org/packages/61/f3/9df65ccb3190d4206402d01c2791b324a01b8cc46de71646a7a9c342ff49/jikanpy-4.2.2.tar.gz`.

**Comment by slashbeast** [comment]
> I'll still fix this, but I'm just curious. Is there a reason you're installing it this way instead of running `pip install jikanpy`? I didn't know people did that and I think it would be good for me to learn why.

I package jinkapy for Gentoo Linux, which uses pypi tarball as the source for installation, for now I've switched to github provided tarballs in https://github.com/gentoo/gentoo/commit/a11a0b54fc27568999b9a7e3a822fe05c4cf4264
